README.md

```diff
@@ -50,3 +50,10 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+To start my code runs through loading an array n long with the numbers before it randomly scrambles them into the starting order so it goes through n twice. then we do the 2-Opt section and we create a new array in 2-Opt so worst case we would reverse the entire tour meaning we have to go through n once and we then need to calculate the distance of each one adding another n. The do while loop runs until it finds 200 improvements or 25 times with no improvement. If we assume the worst case it will use the 200 improvements and take O(200n) time. So the run time will be O(n^2) because of the way that I generate the random order at the start.
```

Comment:
Where does the square come from? I don't see any nested loops that might lead to this.